### DISCUSSION TITLE
Hide an app from the navbar

### DISCUSSION THREAD

**Discussion by Teajo** [grafana/grafana#38369] [Dashboards, Panels & Visualizations]
Hello !

**What would you like to be added**:
It could be great if we can hide an app from the navbar. For example with a toggle located in the app config page.

**Why is this needed**:
This is needed because there are visual bugs when too much apps are instantiated. Moreover It can be great to have the choice to show only most used apps.

Thanks

**Comment by torkelo** [comment]
@ryantxu this seems like a reasonable request. Or have some scroll/collapse mechanism when you have many apps. 

The apps you want to hide, how would you access the app if it was hidden?

**Comment by Teajo** [comment]
> @ryantxu this seems like a reasonable request. Or have some scroll/collapse mechanism when you have many apps.
> 
> The apps you want to hide, how would you access the app if it was hidden?

Maybe with another tab in configuration dropdown or with another sidemenu item which open a page where all apps are listed, as follows : (just an example)
![image](https://user-images.githubusercontent.com/46566099/76195263-3653f400-61e8-11ea-9a75-0afc53c2ddb0.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
